paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/LpmSelectorText.kt

```diff
@@ -3,15 +3,18 @@ package com.stripe.android.paymentsheet.ui
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.unit.sp
-import com.stripe.android.paymentsheet.R
 
 @Composable
-internal fun LpmSelectorText(text: String, modifier: Modifier, isEnabled: Boolean) {
-    val textColor = colorResource(R.color.stripe_paymentsheet_title_text)
+internal fun LpmSelectorText(
+    text: String,
+    textColor: Color,
```

Comment:
Can we just use the textColor from the CompositionLocal rather than pass it in as a parameter?